config/processCcd.py

```diff
@@ -56,12 +56,15 @@
 config.charImage.load(os.path.join(configDir, "cmodel.py"))
 config.charImage.measurement.load(os.path.join(configDir, "apertures.py"))
 config.charImage.measurement.load(os.path.join(configDir, "kron.py"))
+config.charImage.measurement.load(os.path.join(configDir, "hsm.py"))
+if "ext_shapeHSM_HsmShapeRegauss" in config.charImage.measurement.plugins:
+    # no deblending has been done
+    config.charImage.measurement.plugins["ext_shapeHSM_HsmShapeRegauss"].deblendNChild = ""
+    config.charImage.measurement.slots.shape = "base_SdssShape"
```

Comment:
`charImage` is the new stage that maps to the `initialMeasurement` in HSC.  I don't think this is important, but I suggest we just allow HSM to sit in the shape slot because we think it's better than SDSS.